docs/.vitepress/config.ts

```diff
@@ -227,6 +308,7 @@ function prepareSidebar(idx: number) {
 const ogUrl = 'https://vite-plugin-pwa.netlify.app/'
 const ogImage = 'https://vite-plugin-pwa.netlify.app/og-image.png'
 
+// @ts-expect-error avoid TS2321: Excessive stack depth comparing types
```

Comment:
I need to remove this line, it seems on IntelliJ is not working but it works in VSCode.

Reply:
ok, this error seems to appear and dissapear randomly, so we can keep it: https://discord.com/channels/325477692906536972/1012326745753530419/1012347386653319248